### PR TITLE
AKI-646 switch default DB to the rocksDB

### PR DESCRIPTION
**Pull request by AionJayT**
## Description

-

Fixes Issue # .

## Type of change

- [ ] Bug fix.
- [ ] New feature.
- [ ] Enhancement.
- [ ] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

## Testing

-

### CODE REVIEW

File: modAionImpl/src/org/aion/zero/impl/db/DatabaseUtils.java [hunk 1]
```diff
@@ -8,12 +8,17 @@
 import java.nio.file.Paths;
 import java.nio.file.SimpleFileVisitor;
 import java.nio.file.attribute.BasicFileAttributes;
+import java.util.Objects;
 import java.util.Properties;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
```
Comment (AlexandraRoatis) [inline modAionImpl/src/org/aion/zero/impl/db/DatabaseUtils.java]: I think some of these imports are not needed.

File: networks/amity/config.xml [hunk 2]
```diff
@@ -66,9 +66,9 @@
         <state-storage>FULL</state-storage>
         <!--Database implementation used to store data; supported options: leveldb, h2, rocksdb.-->
         <!--Caution: changing implementation requires re-syncing from genesis!-->
-        <vendor>leveldb</vendor>
+        <vendor>rocksdb</vendor>
         <!--Boolean value. Enable/disable database compression to trade storage space for execution time.-->
-        <enable_db_compression>false</enable_db_compression>
+        <enable_db_compression>true</enable_db_compression>
```
Comment (AlexandraRoatis) [inline networks/amity/config.xml]: For LevelDB the compression made performance fairly bad. Do we have experimental data to show that it's better to enable compression with RocksDB?

Comment (AionJayT) [reply networks/amity/config.xml]: I can do the benchmark. But bast on my understanding, Enable the data compression will increase CPU usage but significantly reduce the data size been wrote into the disk, especially the compacting happens. Also I doubt the leveldb library might use the old version of the native levelDB, So the compression performance is not good. for the benchmark I did before. You can look at the AKI-550. 
Also,you can look at the reference link for the details of the rocksdb data compression
https://github.com/facebook/rocksdb/wiki/Compression

Comment (AionJayT) [reply networks/amity/config.xml]: Also, the default setting in rust kernel also enable the data compression.
https://github.com/aionnetwork/aionr/blob/master/db/core/src/dbconfigs.rs#L82-L100

**Pull request merged**